src/PrestaShopBundle/Form/Exception/InvalidConfigurationDataError.php

```diff
@@ -28,6 +28,7 @@
 
 namespace PrestaShopBundle\Form\Exception;
 
+/** @deprecated and will be removed in 9.0 */
```

Comment:
You can call `@trigger_error` before the class to make sure in will appear in the logs.